preon-binding/src/main/java/org/codehaus/preon/annotation/BoundNumber.java

```diff
@@ -76,5 +76,9 @@
      */
     Class<? extends Number> type() default Number.class;
 
+    /**
+     * Helps decide how to do the decoding/encoding based on whether the number is signed or not. Defaults to false (signed).
+     * @return if the value is unsigned.
+     */
     boolean unsinged() default false;
```

Comment:
Only reason I didn't change it was because it was an existing thing, so it could break stuff for anybody moving to the new version. I'm 100% on board with that if you're okay with changing it, though.